test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -115,6 +129,7 @@ public void run() {
             if (Util.compareBufferedImages(imageInit, imageShow)) {
                 ImageIO.write(imageInit, "png", new File("imageInit.png"));
                 ImageIO.write(imageShow, "png", new File("imageShow.png"));
+                ImageIO.write(fullScreen, "png", new File("fullScreenInit.png"));
                 throw new Exception("Failed to show opaque button");
```

Comment:
<del>I suggest moving saving the images into a new method.</del>

I see the set of images is different each time. Probably it makes sense to save all the images which are non-null, what do you think?

Reply:
I don't think it is necessary. And since this is an existing test i'm trying not to change its overall behavior and having screenshots at other stages of the testing is not beneficial in the test debugging.